src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -30,6 +32,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   }
 
   @Override
+  public void onStart() {
```

Comment:
That's too late. It has to be done earlier.
On 05-Mar-2015 4:29 pm, "Abhideep Singh" <notifications@github.com> wrote:

> In src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java
> <https://github.com/sarality/appblocks/pull/267#discussion_r25855389>:
>
> > @@ -30,6 +32,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
> >    }
> >
> >    @Override
> > +  public void onStart() {
>
> why onStart? Why not in render? If we are worried about render being
> overriden make that a public final that calls a doRender which can be
> overriden and then render can call ActionManager.setup
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sarality/appblocks/pull/267/files#r25855389>.
>